pkg/config/sink.go

```diff
@@ -168,15 +168,18 @@ func (s *SinkConfig) applyParameter(sinkURI *url.URL) error {
 	}
 
 	// validate that protocol is compatible with the scheme
-	if isMqScheme(sinkURI.Scheme) {
+	if IsMqScheme(sinkURI.Scheme) {
```

Comment:
Please refine this comment to start with `IsMqScheme`.

Reply:
Ref: https://github.com/pingcap/tiflow/pull/6461/files#diff-1aa9ba3321cb0edb368a32b5084da34322b0f9d6a600019beb724ab63bc5e071R191